packages/kotti-ui/source/kotti-filter/components/Search.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+	<KtFieldText
+		hideValidation
+		:isLoading="isLoading"
+		:leftIcon="Yoco.Icon.SEARCH"
+		:placeholder="placeholder"
+		:size="Kotti.Field.Size.SMALL"
+		:value="searchValue"
+		@input="handleSetSearchValue"
+	/>
+</template>
+
+<script lang="ts">
+import { KtFieldText } from '@3yourmind/kotti-ui'
+import { Yoco } from '@3yourmind/yoco'
+import { computed, defineComponent } from '@vue/composition-api'
+
+import { useTranslationNamespace } from '../../kotti-translation/hooks'
+import { Kotti } from '../../types'
+import { getSearchFilterInitialState } from '../utils'
+
+export default defineComponent<{
+	column: Kotti.Filter.Column
+	filter: Kotti.Filter.Filter | null
+	isLoading: boolean
+}>({
+	name: 'Search',
+	components: {
+		KtFieldText,
+	},
+	props: {
+		column: {
+			required: true,
+			type: Object,
+		},
+		filter: {
+			default: null,
```

Comment:
What’s the use-case for a `default` `filter?

Reply:
When the KtFilter value does not have any value for the global search, then there's nothing to pass to Search in filter prop

Reply:
which shouldn't render the component at all
i think this pattern is repeated across different components

usually the prop that is the driving feature for even rendering the component should be required, and the component wouldn't render otherwise